pom.xml

```diff
@@ -68,99 +68,45 @@
     </repositories>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>structs</artifactId>
+            <version>1.10</version>
+        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.29.1</version>
+            <version>1.40</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-step-api</artifactId>
-            <version>${workflow.version}</version>
+            <version>2.14</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>${workflow.version}</version>
+            <version>2.41</version>
```

Comment:
Should this not be a `test`-scoped dep? If you are actually depending on it, something is very wrong.

Reply:
`ReadMavenPomStep` implements a `GroovyShellDecorator`

Reply:
> ReadMavenPomStep implements a GroovyShellDecorator

 Here There Be Dragons

Reply:
I will fix this.

Reply:
#38